app/controllers/courses_controller.rb

```diff
@@ -24,6 +24,15 @@ def destroy
     head :no_content
   end
 
+  # get visible returns courses that are taught by an admin or enrolled in by a student
+  def get_visible
+    if current_user.admin
```

Comment:
We could make this one line:

`render status: :ok, json: current_user.taught_courses if current_user.admin else current_user.enrolled_courses`